### ISSUE TITLE
How to open the 2nd door?

### ISSUE THREAD

**Issue by voelkerb** [ubilab-ws21/esclab#12]
We still need to figure this out. Do we need to input the code via plasma globes?
If yes, how to input 4 digit code using 2 people?
Should we install a keypad somewhere?

I can reprogram the plasma globes but they can only binary detect a touch.

**Comment by sboettcher** [comment]
Same as #4

**Comment by sboettcher** [comment]
Might be a bit un-obvious to players, but how about just using the same keypad as for door one again? Could be the easiest to implement since we need to touch that anyway.

**Comment by voelkerb** [comment]
Yeah - agree - somehow unobvious. If this is broken (which I think it is), we have to build a new one anyway, so maybe directly build two? But then the participants may still ask why the globes are installed at all...

**Comment by sboettcher** [comment]
We could just label the keypad with something like "access to automatic doors" to make it more obvious.
Also there is a bunch of other stuff not used as well...
Sure if new keypads need to be built anyway then might as well make two.

**Comment by pscholl** [comment]
Think we talked about this last time. Knock-Knock will be the lock to the 2nd door.

It will be activated after the radio puzzle is complete, which will then show the current time. The current time is the input to the knock-knock puzzle.

**Comment by sboettcher** [comment]
@pscholl what's the current status here again? do the relevant groups know about it?
